package/MDAnalysis/core/universe.py

```diff
@@ -491,10 +491,11 @@ def transfer_to_memory(self, start=None, stop=None, step=None,
             # Overwrite trajectory in universe with an MemoryReader
             # object, to provide fast access and allow coordinates
             # to be manipulated
+            if step is None: step=1 
```

Comment:
minor thing: we typically use separate lines for blocks:
```python
if step is None:
   step = 1
```

And we also follow [PEP 8](https://www.python.org/dev/peps/pep-0008/), which means spaces between operators
```python
step = 1
```

unless they are assignment operators in kwargs.